### DISCUSSION TITLE
How to improve accuracy of model

### DISCUSSION THREAD

**Discussion by rpassaro** [kyleskom/NBA-Machine-Learning-Sports-Betting#393] [Q&A] [answered]
Hi! Super quick question: If I keep running the commands to train the model, will the accuracy of the model improve? For example, will: XGBoost_53.7%_UO-9.json  XGBoost_54.8%_UO-8.json maybe change to XGBoost_56% and 56.2%? Is that possible? Just wondering, thanks!

**Comment by nealmick** [comment, marked as answer]
It is certainly possible.  Just keep in mind that no model is infallible and evaluation docent always translate to future games.  I recommend trying to understand how models are trained and how data is formed.  Arbitrage and EV market consensus strategies work with some time and effort but it isnt unlimited.

**Reply by STRATZ-Ken** [reply]
`Arbitrage and EV market consensus strategies work with some time and effort but it isnt unlimited.`

Winning isnt unlimited. Regardless of the form.

**Reply by nealmick** [reply]
yup, sports gambling is fixed market.  only can place bets if bookies let you.  winners are not allowed to play.  Arbing currency and commodities on large exchange may have less limits.   can definitely make good money for a few months arbing sports but limits are inevitable.
